apps/documentation/src/routes/documentation/tutorial/redirections.mdx

```diff
@@ -13,13 +13,11 @@ import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
 # Redirections
 
-## Handle redirections
+What if there is a Pokémon among all of them that should be considered the GOAT? What
+we are going to do right now is create a new route `/pokemons/GOAT` that points to the best
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The plural of ‘Pokémon’ is “Pokémon”. (POKEMONS[1])
Suggestions: `Pokémon`
URL: https://en.wikipedia.org/wiki/Pok%C3%A9mon#Name 
Rule: https://community.languagetool.org/rule/show/POKEMONS?lang=en-US&subId=1
Category: GRAMMAR
<!-- __reviewdog__:ChA3MjI5YTZmOGYzOWIxOWU1EgxMYW5ndWFnZVRvb2w= -->